### ISSUE TITLE
CVE-2021-4122

### ISSUE THREAD

**Issue by shinsenter** [serversideup/docker-php#44]
I scanned the `php:8.1-fpm-apache` version and it said that there is a HIGH level vulnerability found.

Package: cryptsetup:2:2.2.2-3ubuntu2.3
Level: HIGH
Description: [disable encryption via header rewrite]

https://ubuntu.com/security/CVE-2021-4122

**Comment by jaydrogers** [comment]
Thanks for the report. Just for a future note, we do have our security details listed in [SECURITY.md](https://github.com/serversideup/docker-php/blob/main/SECURITY.md) which links to [our responsible disclosure policy](https://www.notion.so/Responsible-Disclosure-Policy-421a6a3be1714d388ebbadba7eebbdc8).

Reporting vulnerabilities in a private conversation helps protect other users and gives us a chance to disclose vulnerabilities after they already have been patched.

Since the conversation is already public, I am okay with continuing to work in the open to resolve this issue.

Our usual template for reporting a vulnerability is like this:
```
# Summary
...

# Vulnerability details
...

# Impact
...

# Proof of concept
...

# Mitigation
...
```

I also looked at the link you provided and noticed:

* The vulnerability was downgraded to "MEDIUM"
* If I am reading the link correctly, the patch is not yet available for Ubuntu 20.04? See the image below:
<img width="1099" alt="Screen Shot 2022-01-21 at 9 36 19 AM" src="https://user-images.githubusercontent.com/3174134/150555927-3308be7c-3b08-4746-bd8e-1f4f552463cb.png">

# Moving forward
1. What scanner did you use so I can verify it's no longer vulnerable?
2. Do you know of any other mitigation steps other than just updating?

**Comment by shinsenter** [comment]
Hey @jaydrogers, thank you for the responding.
I used AWS scanner. But I don't really know the details of this.
Actually I need to explain to my boss about the vulnerabilities in order to be allowed to use your image.
